components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/config/SCIMProviderDAO.java

```diff
@@ -119,9 +119,8 @@ public SCIMProviderDTO getProvider(String consumerId, String providerId)
             throw new IdentitySCIMException(errorMsg, e);
         } catch (SQLException e) {
             log.error("Error when executing the SQL : " + SQLQueries.GET_PROVIDER_SQL);
```

Comment:
Why log SQL queries. I think these should be debug logs. Otherwise these logs disclose failed queries which is an internal details of the server